.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contribution to Deprecation Toolkit
+
+## Releasing a new version
```

Comment:
This project is open source, and only a small fraction of contributors can release, so this documentation doesn't belong to the CONTRIBUTING.md file. We can change a RELEASE.md file in the root though.